editor/src/core/shared/template-path.ts

```diff
@@ -647,6 +669,38 @@ export function isAncestorOf(
   }
 }
 
+function fullElementPathForPath(path: TemplatePath): ElementPath[] {
+  if (isScenePath(path)) {
+    return path.sceneElementPaths
+  } else {
+    return [...path.scene.sceneElementPaths, path.element]
+  }
+}
+
+export function isDescendantOf(
```

Comment:
there is an `isAncestorOf` in the template-path.ts, should that be replaced with this one?